doc/source/contributing.rst

```diff
@@ -211,30 +194,17 @@ for switching from viewing the documentation for the latest stable release
 to viewing the documentation for the development version or previously
 released versions.
 
-Code style
-----------
-
-As indicated in `Coding style <https://dev.docs.pyansys.com/coding-style/index.html>`_
-in the *PyAnsys Developer's Guide*, PyMechanical follows PEP8 guidelines. PyMechanical
-implements `pre-commit <https://pre-commit.com/>`_ for style checking.
-
-To ensure your code meets minimum code styling standards, run these commands::
-
-  pip install pre-commit
-  pre-commit run --all-files
-
-You can also install this as a pre-commit hook by running this command::
+Post issues
+-----------
 
-  pre-commit install
+Use the `PyMechanical Issues <https://github.com/ansys/pymechanical/issues>`_
+page to submit questions, report bugs, and request new features. When possible,
+use these templates:
 
-This way, it's not possible for you to push code that fails the style checks::
+* Bug report
+* Feature request
```

Comment:
Add links to new issue creation based on these templates. This will make things easier for end users.